src/app/shared/services/switchboard-toaster.service.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { TestBed } from '@angular/core/testing';
+
+import { SwitchboardToasterService } from './switchboard-toaster.service';
+import { ToastrModule, ToastrService } from 'ngx-toastr';
+
+describe('SwitchboardToasterService', () => {
+  let service: SwitchboardToasterService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [ToastrService],
+      imports: [ToastrModule.forRoot()]
+    });
+    service = TestBed.inject(SwitchboardToasterService);
+  });
+
+  it('should be created', () => {
```

Comment:
It would be great if you add some tests to this service, so in future nobody break this by refactor/adding new functionalities.